.mergify.yml

```diff
@@ -95,14 +99,14 @@ pull_request_rules:
       delete_head_branch:
   - name: automatic merge when CI passes for the make update-beats
     conditions:
-      - check-success=apm-ci/pr-merge
       - label=automation
       - base~=^update-beats
       - files~=^(go.mod|go.sum|NOTICE.txt)
+      - check-success=apm-ci/pr-merge
```

Comment:
ditto

Reply:
moved - check-success=beats-ci/pr-merge to the begin of the list